doc/cli/vast-import-syslog.md

```diff
@@ -0,0 +1,6 @@
+The syslog import format uses [RFC5424](https://tools.ietf.org/html/rfc5424) consumes syslog messages in format defined in RFC 5424. 
+
+In order to run consume syslog messages use the following command:
+
+```
+vast import syslog -r <path-to-syslog-file>
```

Comment:
```suggestion
Ingest Syslog messages into VAST. The following formats are supported:

- [RFC 5424](https://tools.ietf.org/html/rfc5424)
- A fallback format that consists only of the Syslog message.

If you require support for any other Syslog formats, feel free to contact us at
info@tenzir.com.

Usage example:

```sh
# Import from file.
vast import syslog -r path/to/sys.log

# Continuously import from a stream.
syslog | vast import syslog
```
```

Reply:
I have a question related to this suggestion. Shall I mention that the command `syslog | vast import syslog` interprets each message as one string for now?